README.md

```diff
@@ -26,6 +27,10 @@ Steps to run the project in PC:
 
     ``` npm install http-server -g ```
 
+    Mac and Linux users should add suo in the  beginning:
```

Comment:
There is no need to use `sudo` to install `http-server`. In general any webserver will do, but `http-server` is a good option.